ssas/service/admin/api.go

```diff
@@ -0,0 +1,40 @@
+package admin
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/CMSgov/bcda-app/ssas"
+	"net/http"
+)
+
+func createSystem(w http.ResponseWriter, r *http.Request) {
+	type system struct {
+		ClientName string `json:"client_name"`
+		GroupID    string `json:"group_id"`
+		Scope      string `json:"scope"`
+		PublicKey  string `json:"public_key"`
+		TrackingID string `json:"tracking_id"`
+	}
+
+	sys := system{}
+	if err := json.NewDecoder(r.Body).Decode(&sys); err != nil {
+		http.Error(w, "Could not create system due to invalid request body", http.StatusBadRequest)
+		return
+	}
+
```

Comment:
Could you put in a call to ssas.OperationCalled?:
`ssas.OperationCalled(ssas.Event{Op: "RegisterClient", TrackingID: TrackingID})`

Reply:
Adding that now. Should there also be `OperationFailed` on errors and `OperationSucceeded` if everything works as expected?